Sources/GenericModule/View/View.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright © 2020 Rosberry. All rights reserved.
+//
+
+public protocol View: class {
+    associatedtype Output
+    associatedtype ViewModel: GenericModule.ViewModel
+    associatedtype ViewInput
+
+    var output: Output? { get set }
+
+    init(viewModel: ViewModel)
```

Comment:
What do you think about passing Output to init and View.output become not optional? Is it possible?

Reply:
view output is not optional now